packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -129,6 +130,17 @@ class BuildInfo {
   /// rerun tasks.
   final String? performanceMeasurementFile;
 
+  /// Configure a constant pool file.
+  /// Additional constant values to be made available in the Dart program.
+  ///
+  /// These values can be used with the const `fromEnvironment` constructors of
+  ///  [String],the key is json field,the value is
+  /// json value
+  ///
+  /// An additional field `defineConfigJsonRawValue` is provided to represent the native JSON value of the configuration file
```

Comment:
You already remove this right? the comment should be updated properly

Reply:
yes, i'll correct it